extensions/ql-vscode/src/remote-queries/remote-queries-manager.ts

```diff
@@ -81,6 +89,26 @@ export class RemoteQueriesManager {
     }
   }
 
+  public async autoDownloadRemoteQueryResults(
+    queryResult: RemoteQueryResult,
+    token: CancellationToken
+  ): Promise<void> {
+    const analysesToDownload = queryResult.analysisSummaries
+      .filter(a => a.fileSizeInBytes < autoDownloadMaxSize)
+      .slice(0, autoDownloadMaxCount)
+      .map(a => ({
+        nwo: a.nwo,
+        resultCount: a.resultCount,
+        downloadLink: a.downloadLink,
+        fileSize: a.fileSizeInBytes.toString()
```

Comment:
In general, slightly safer to do this in order to handle `undefined` and `null` without erroring.

```suggestion
        fileSize: String(a.fileSizeInBytes)
```